src/cli.js

```diff
@@ -18,10 +18,8 @@ export function terminalWidth(_process=process) {
 }
 
 export default argv
-  // jscs:disable
```

Comment:
I just casually saw this, but we could/should probably do a grep and see if there are other buried `// jscs` comments lurking.